deploy/1_deploy.ts

```diff
@@ -1,21 +1,61 @@
-import { HardhatRuntimeEnvironment } from 'hardhat/types';
 import { DeployFunction } from 'hardhat-deploy/types';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { buildMerkleTree } from '../src';
 
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
-  const { deployments, getNamedAccounts, ethers } = hre;
-  const { deploy, log } = deployments;
-
-  const { deployer } = await getNamedAccounts();
-
-  const merkleFunder = await deploy('MerkleFunder', {
-    from: deployer,
+  const { deployments, getUnnamedAccounts, ethers } = hre;
+  const { address: merkleFunderAddress, abi: merkleFunderAbi } = await deployments.deploy('MerkleFunder', {
+    from: (await getUnnamedAccounts())[0],
```

Comment:
Just curious, is it better to getUnnamedAccounts over getNamedAccounts?

Reply:
getNamedAccounts() feels like an additional level of abstraction to go wrong to me